stdlib/auth/src/main/ballerina/src/auth/inbound_auth_provider.bal

```diff
@@ -18,9 +18,9 @@
 # should be object-wise similar.
 public type InboundAuthProvider abstract object {
 
-    # Authenticate with credential value passed.
+    # Authenticate with the credential value passed as a `string`.
     #
     # + credential - Credential value
-    # + return - True if authentication is a success, else false or `Error` if any error occurred
+    # + return - `true` if authentication is successful, else `false` or else an `Error` in case of an error
```

Comment:
```suggestion
    # + return - `true` if authentication is successful, otherwise fails, or else an `Error` in case of an error
```
Better to specify the type of the error.